src/pii-client.ts

```diff
@@ -0,0 +1,79 @@
+import PIIsdk, {
+    AgentType,
+    initAgent,
+    PIIEncryptionMethod,
+} from "@notabene/pii-sdk";
+import { TransactionArguments, TravelRuleOptions } from "./types";
+
+export class PIIEncryption {
+    private readonly config: TravelRuleOptions;
+    public toolset: PIIsdk;
+
+    constructor(config: TravelRuleOptions) {
+        this.config = config;
+        const requiredFields = [
+            "kmsSecretKey",
+            "baseURLPII",
+            "audiencePII",
+            "clientId",
+            "clientSecret",
+            "authURL",
+            "jsonDidKey",
+        ];
```

Comment:
Please move this variable to be a global constant

Reply:
Removed